privacyidea/app.py

```diff
@@ -196,38 +171,37 @@ def create_app(config_name="development",
 
     app.response_class = PiResponseClass
 
+    # Setup logging
     try:
         # Try to read logging config from file
         log_config_file = app.config.get("PI_LOGCONFIG",
                                          "/etc/privacyidea/logging.cfg")
         if os.path.isfile(log_config_file):
-            logging.config.fileConfig(log_config_file)
+            _, ext = os.path.splitext(log_config_file)
+            if ext.upper() in ['.YML', '.YAML']:
+                log_cnf_dict = yaml.safe_load(open(log_config_file, 'r').read())
+                logging.config.dictConfig(log_cnf_dict)
+            else:
+                logging.config.fileConfig(log_config_file)
             if not silent:
-                print("Reading Logging settings from {0!s}".format(log_config_file))
+                sys.stdout.write("Read Logging settings from {0!s}".format(log_config_file))
```

Comment:
Did you mean ``stderr`` or what is the reason to change print or write?

Reply:
Fixed in 9136a076